### PR TITLE
scheduler: elastic quota ignore terminating pod

### PR DESCRIPTION
**Pull request by shaloulcy**
### Ⅰ. Describe what this PR does

pod is deleted gracefully, it may be stuck long time. some user want ignore the pods when using elastic quota

### V. Checklist

- [ ] I have written necessary docs and comments
- [ ] I have added necessary unit tests and integration tests
- [ ] All checks passed in `make test`

### CODE REVIEW

File: pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go [hunk 1]
```diff
@@ -1054,3 +1084,19 @@ func (gqm *GroupQuotaManager) doUpdateOneGroupSharedWeightNoLock(quotaName strin
 
 	gqm.updateOneGroupSharedWeightNoLock(quotaInfo)
 }
+
+func isTerminatingPod(pod *v1.Pod) bool {
```
Comment (hormes) [inline pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go:1088]: It is strange to add feature-gate in isTerminatingPod. Why not change the name of this function to shouldBeIgnored and also change the previous related functions.

Comment (shaloulcy) [reply pkg/scheduler/plugins/elasticquota/core/group_quota_manager.go]: fix it

**Review by saintube — APPROVED** [review]
/lgtm

**Pull request merged**